driver-legacy/src/main/com/mongodb/MongoClientOptions.java

```diff
@@ -392,6 +398,14 @@ public int getMaxConnecting() {
         return maxConnecting;
     }
 
+    public long getMaintenanceInitialDelay() {
```

Comment:
Javadoc

Reply:
I think you were looking at an earlier commit -- I ran static checks after pushing commit 1 and added a Javadoc comment. No `@since` comment, though.